connectors/v2/myradio.ua.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '.cur-blk';
+
+Connector.separators = [' -             '];
```

Comment:
Not sure that this is the best way. Maybe we need to apply .trim() to artist and track in the original function?